CHANGELOG.md

```diff
@@ -10,6 +10,14 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## [Unreleased]
 
+- 🐞 The example configuration file contained an invalid section `vast`.
+   This has been changed to the correct name `system`.
+
+- 🔄 Two minor modifications were done in the parsing framework:
+  - The parsers for enums and records now allow trailing separators.
+  - The dash ('-') was removed from the allowed characters of schema type
+    names.
```

Comment:
The nesting doesn't look appealing to me when inspecting the log. Let's use plain paragraph style:

```markdown
- 🔄 Two minor modifications were done in the parsing framework: (i) the
  parsers for enums and records now allow trailing separators, and (ii) the
  dash (`-`) was removed from the allowed characters of schema type names.
  [#706](https://github.com/tenzir/vast/pull/706
```